lib/xray_creator.py

```diff
@@ -210,26 +217,29 @@ def _find_device_books(self, book_lookup, log):
                 if book_lookup.has_key(book._data['uuid']):
                     book_id = book_lookup[book._data['uuid']].book_id
                     fmt = book.path.split('.')[-1].lower()
-                    if (fmt != 'mobi' and fmt != 'azw3') or (fmt == 'mobi' and 'mobi' not in self._formats) or (fmt == 'azw3' and 'azw3' not in self._formats):
+                    if ((fmt != 'mobi' and fmt != 'azw3') or
+                            (fmt == 'mobi' and 'mobi' not in self._formats) or
+                            (fmt == 'azw3' and 'azw3' not in self._formats)):
                         continue
                     books[book_id][fmt] = {'device_book': book.path,
-                        'device_sdr': '.'.join(book.path.split('.')[:-1]) + '.sdr'}
+                                           'device_sdr': '.'.join(book.path.split('.')[:-1]) + '.sdr'}
                     self._num_of_formats_found_on_device += 1
             return books
-        except (TypeError, AttributeError) as e:
+        except (TypeError, AttributeError):
             self._num_of_formats_found_on_device = -1
-            log('%s Device found but cannot be accessed. It may have been ejected but not unplugged.' % datetime.now().strftime('%m-%d-%Y %H:%M:%S'))
+            log(('{0} Device found but cannot be accessed. '
+                 'It may have been ejected but not unplugged.').format(datetime.now().strftime('%m-%d-%Y %H:%M:%S')))
             return None
-        except Exception as e:
+        except Exception as msg:
```

Comment:
IIRC most python documentation uses 'e', which is perhaps just a failure on their part to pick a sensible name in documentation :)  I would argue this could be an allowed short named variable without problems though.